src/message.coffee

```diff
@@ -128,68 +153,93 @@ class SlackTextMessage extends TextMessage
             parts.push(text.slice(cursor, result.index), m)
 
         else
-          link = link.replace /^mailto:/, ''
+          link = link.replace /^mailto:/, ""
           if label and -1 == link.indexOf label
             parts.push(text.slice(cursor, result.index), "#{label} (#{link})")
           else
             parts.push(text.slice(cursor, result.index), link)
 
       cursor = regex.lastIndex
-      if (result[0].length == 0) 
+      if (result[0].length == 0)
         regex.lastIndex++
 
     parts.push text.slice(cursor)
 
-    return Promise.all(parts).then((substrings) ->
-      return substrings.join('')
-    )
+    return Promise.all(parts)
+      .then (substrings) ->
+        return substrings.join("")
 
   ###*
-  # Returns name of user with id
+  # Creates a mention from a user ID
+  #
+  # @private
+  # @param {SlackClient} client - a client that can be used to get more data needed to build the text
+  # @param {string} id - the user ID
+  # @param {Array<Mention>} mentions - a mentions array that is updated to include this user mention
+  # @returns {Promise<string>} - a string that can be placed into the text for this mention
   ###
   replaceUser: (client, id, mentions) ->
-    client.web.users.info(id).then((res) =>
-      if res?.user?
-        user = res.user
-        mention = new SlackMention(user.id, 'user', user)
-        mentions.push(mention)
-        return "@#{user.name}"
-      else return "<@#{id}>"
-    )
-    .catch((error) =>
-      client.robot.logger.error "Error getting user info #{id}: #{error.message}"
-      return "<@#{id}>"
-    )
+    client.web.users.info(id)
+      .then (res) =>
+        if res?.user?
+          user = res.user
+          mention = new SlackMention(user.id, "user", user)
```

Comment:
To match styling in `replaceLinks()`, might want to change to `mentions.push new SlackMention(user.id, "user", user)`